src/lava/magma/core/run_configs.py

```diff
@@ -67,42 +67,60 @@ def select(self,
         pass
 
 
-# ToDo: This is only a minimal RunConfig that will select SubProcModel if there
-#       is one and if not selects a PyProcessModel with the correct tag.
-#       Needs to be modified in future releases to support more complicated
-#       sync domains, @requires (example: GPU support), and to select
-#       LeafProcModels of type other than PyProcessModel.
 class Loihi1SimCfg(RunConfig):
     """Run configuration selects appropriate ProcessModel -- either
-    SubProcessModel for a Hierarchical Process or else a PyProcessModel for a
-    standard Process. The appropriate PyProcessModel is selected based @tag(
-    'floating_pt') or @tag('fixed_pt'), for floating point precision or Loihi
-    bit-accurate fixed point precision respectively"""
+    `SubProcessModel` for a hierarchical Process or else a `PyProcessModel`
+    for a standard Process.
 
-    def __init__(self, custom_sync_domains=None, select_tag='floating_pt',
-                 select_sub_proc_model=False):
+    If any `SubProcessModel`s are available, then they will be returned first.
+
+    In case of `PyProcessModel`s, an appropriate `PyProcessModel` is selected
+    based on the tags associated with it. Tags are set using the `@tag`
+    decorator. If no tags are set for `ProcessModel`s and no tag is selected,
+    then the first matching `ProcessModel` will be returned.
+
+    If any exceptions are needed, they can be specified in
+    `exception_proc_model_map` dictionary as `{Process: ProcessModel}`
+    key-value pairs. These explicit specifications are given
+    precedence over the subsequent logic for `ProcessModel` selection"""
```

Comment:
I think, adding method descriptions would be helpful, e.g. for select(..)

Reply:
I have rearranged the class docstring